wagtailio/project_styleguide/templates/patterns/pages/blog/blog_index_page.html

```diff
@@ -0,0 +1,47 @@
+{% extends "patterns/base_page.html" %}
+
+{% load static wagtailcore_tags wagtailadmin_tags wagtailimages_tags wagtailio_utils %}
+
+{% block body_class %}template-blog{% endblock %}
+
+{% block content %}
+
+    <div class="container">
+        <div class="hero">
+            <div class="hero__inner">
+                <h1 aria-hidden="true">Latest Chirps</h1>
+                <span class="u-sr-only">Latest blog posts</span>
+            </div>
+        </div>
+
+        <section class="blog-listing">
+            {% for post in posts %}
+                {% if forloop.first or forloop.counter|divisibleby:6 %}
+                    {% include 'patterns/pages/blog/includes/blog_post.html' with post=post featured=True %}
```

Comment:
It doesn't look like you've done anything that should affect the hover states. I'm getting inconsistent results when they are active or not. Perhaps there's an issue with a tag here.

https://user-images.githubusercontent.com/22497837/186181071-a1091235-e7a7-465f-9bf2-27cf6c63d6b0.mp4

Reply:
Paul found this in QA and it seems to be a bug when viewing inside the PL but is ok on the frontend of the site. I did have a look when he reported it but couldn't pin it down.